core/src/kvs/ds.rs

```diff
@@ -508,12 +509,37 @@ impl Datastore {
 
 	// Initialise the cluster and run bootstrap utilities
 	#[instrument(err, level = "trace", target = "surrealdb::core::kvs::ds", skip_all)]
-	pub async fn bootstrap(&self) -> Result<(), Error> {
-		// Insert this node in the cluster
-		self.insert_node(self.id).await?;
-		// Mark expired nodes as archived
-		self.expire_nodes().await?;
-		// Everything ok
+	pub async fn get_version(&self) -> Result<StorageVersion, Error> {
+		let tx = self.transaction(TransactionType::Write, LockType::Pessimistic).await?.enclose();
+		let keys = catch!(tx, tx.keys(crate::key::storage::version::suffix()..vec![0xff], 1));
+
+		let version = if keys.is_empty() {
+			StorageVersion::latest()
+		} else {
+			let key = crate::key::storage::version::new();
+			let num = match catch!(tx, tx.get(key, None)) {
+				Some(v) => u16::from_ne_bytes(v.as_slice().to_owned().try_into().unwrap()),
+				None => 0 as u16,
+			};
+			num.into()
+		};
+
+		let num: u16 = version.clone().into();
+		let bytes = num.to_ne_bytes().to_vec();
```

Comment:
nit: maybe we should explicitly pick either big endian or little endian here instead of native endian which depends on the platform?

Reply:
In other places we are using BE - I suspect because that's the network format and therefore the one that the `bytes` crate uses, so `to/from_be_bytes` should be fine.

Reply:
Cool! I'll make those changes

Reply:
Addressed in [a1fafdd](https://github.com/surrealdb/surrealdb/pull/4581/commits/a1fafdd4246f9a795c096204ff6750a6a6a571b9)